Geometry/TrackerNumberingBuilder/plugins/TrackerGeometricDetESModule.cc

```diff
@@ -24,9 +24,30 @@ using namespace edm;
 
 TrackerGeometricDetESModule::TrackerGeometricDetESModule( const edm::ParameterSet & p ) 
   : fromDDD_( p.getParameter<bool>( "fromDDD" )),
-    layerNumberPXB_( p.exists( "layerNumberPXB" ) ? p.getParameter<unsigned int>( "layerNumberPXB" ) : 16U ),// 16 for current, 18 for SLHC 
-    totalBlade_( p.exists( "totalBlade" ) ? p.getParameter<unsigned int>( "totalBlade" ) : 24U )             // 24 for current, 56 for SLHC 
```

Comment:
In any case the fix should be such not to use those two parameters in the configurations anymore. I thought fixing the configurations in fillDescriptions was enough. Were are those parameters still used?